### ISSUE TITLE
ACLs not loaded with module

### ISSUE THREAD

**Issue by dstags-LO** [openlabs/magento_integration#96]
Logging into OpenERP with user other than admin results in numerous stack traces relating to ACLs throughout application. 

Research indicates a need to create a  /security folder containing ir.model.access.csv and associated xml files for setting up model permissions for user groups.

See example stack below when attempting to access a simple product.

2013-07-29 14:58:12,072 23661 WARNING oerpdev openerp.addons.base.ir.ir_model: Access Denied by ACLs for operation: read, uid: 3, model: product.price_tier
2013-07-29 14:58:12,073 23661 ERROR oerpdev openerp.netsvc: Access Denied
Sorry, you are not allowed to access this document. Please contact your system administrator if you think this is an error.

(Document model: product.price_tier)
Traceback (most recent call last):
  File "/opt/openerp/server/openerp/netsvc.py", line 292, in dispatch_rpc
    result = ExportService.getService(service_name).dispatch(method, params)
  File "/opt/openerp/server/openerp/service/web_services.py", line 626, in dispatch
    res = fn(db, uid, _params)
  File "/opt/openerp/server/openerp/osv/osv.py", line 188, in execute_kw
    return self.execute(db, uid, obj, method, *args, *_kw or {})
  File "/opt/openerp/server/openerp/osv/osv.py", line 144, in wrapper
    raise except_osv(inst.name, inst.value)
except_osv: (u'Access Denied', u'Sorry, you are not allowed to access this document. Please contact your system administrator if you think this is an error.\n\n(Document model: product.price_tier)')
2013-07-29 14:58:12,074 23661 INFO oerpdev werkzeug: 192.168.37.88 - - [29/Jul/2013 14:58:12] "POST /web/dataset/call_kw HTTP/1.1" 200 -

**Comment by shalabhaggarwal** [comment]
@dstags-LO There are no extra ACLs to be provided along with the module.

All what is required is that the user who is supposed to access the magento related stuff on openerp should belong to `base.group_user`. See here https://github.com/openlabs/magento_integration/blob/develop/magento.xml#L5

**Comment by l-arnold** [comment]
There is no base.user.group definied in my system.  Magento works fine for Admin, but I cannot assign my normal user self.

Magento does not seem to be an Application defined in the OpenERP set of objects that can have rights assigned to it in OpenERP.  Still Magento does show up as Menu Item for Admin and most wizards can be launched.  Not so easy with "finishing"

**Comment by omidkosari** [comment]
@shalabhaggarwal  I don't know why but magento related permission loaded in "Human Resources / Employee"  group which is one of predefined OpenERP default groups .

**Comment by coruble** [comment]
I know this is an old issue but I ran into it and it made me go nuts... None of the solutions suggested above worked for me (i.e. I had no "base.group_user" group, and the user was already part of all required groups...). I finally found a solution.

I added this line to "ir.mode.access.csv" (https://github.com/openlabs/magento_integration/blob/develop/security/ir.model.access.csv)

access_product_price_tier,access_product_price_tier,model_product_price_tier,base.group_sale_salesman,1,0,0,0

Works great with a user who's not "admin".

OpenLabs guys, maybe consider adding this ACL to your access rights file ?
